scripts/create-pv.sh

```diff
@@ -22,11 +22,32 @@ for name in $released; do
     oc patch pv -p '{"spec":{"claimRef": null}}' $name
 done
 
-NODE_NAMES=$(oc get node -o name -l node-role.kubernetes.io/worker)
-if [ -z "$NODE_NAMES" ]; then
-    echo "Unable to determine node name with 'oc' command."
-    exit 1
+# OCP 4.18+
+ssh_key="${HOME}/.crc/machines/crc/id_ed25519"
+
+# OCP 4.17 and earlier
+if [ ! -f "${ssh_key}" ]; then
+    ssh_key="${HOME}/.crc/machines/crc/id_ecdsa"
+fi
+
+if [ -f "${ssh_key}" ]; then
+    NODE_IPS=$(oc get nodes -o template --template '{{range .items}}{{range .status.addresses}}{{if eq .type "InternalIP"}}{{.address}}{{"\n"}}{{end }}{{end }}{{end }}')
+    if [ -z "$NODE_IPS" ]; then
+        echo "Unable to determine node IPs with 'oc' command."
+        exit 1
+    fi
+    echo "Using SSH key located at ${ssh_key} for PV creation"
+    for node in $NODE_IPS; do
+        ssh -i "${ssh_key}" -o StrictHostKeyChecking=no core@"${node}" "for i in `seq -w -s ' ' $PV_NUM`; do echo \"creating dir /mnt/openstack/pv\$i on $node\"; mkdir -p /mnt/openstack/pv\$i; done"
+    done
+else
+    NODE_NAMES=$(oc get node -o name -l node-role.kubernetes.io/worker)
```

Comment:
ok basically this will be executed when we're in an environment != crc and in case we do not find any key, which preserves compatibility w/ other development workflows.